Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/AutoLeftRight.java

```diff
@@ -0,0 +1,24 @@
+package org.usfirst.frc.team5493.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class AutoLeftRight extends CommandGroup {
+
+    public AutoLeftRight() {
+       //addSequential(new IntakeRelease());
+    	addSequential(new DriveStraightWithGyro());
+    	//addSequential(new TurnWithGyro());
+    	addSequential(new DriveStraightWithGyro());
+    	//addSequential(new TurnWithGyro());
+    	addSequential(new DriveStraightWithGyro());
+    	addSequential(new ClimbUp());
+    	addSequential(new ClimbHold());
+    	addSequential(new DriveStraightWithGyro());
+    	addSequential(new CubeOuttake());
+    	//addSequential(new DriveStraightBack());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)